iree/testing/vulkan/CMakeLists.txt

```diff
@@ -24,6 +22,21 @@ endif()
 # If Vulkan is not found, omit the target.
 include(FindVulkan)
 if(NOT Vulkan_FOUND)
+  message("Could not find Vulkan, disabling Vulkan GUI programs")
```

Comment:
do these print all the time? like, will this be our equivalent to tf's "NO CUDA AND WHATEVER INTEL MKL MUMBLE FOUND OMG PANIC but actually just a warning" spam?

Reply:
When `IREE_BUILD_SAMPLES` is enabled (it is by default), yes. I'd like to leave _some_ indication of why targets are not defined. Any ideas on how to make this quieter for people that don't care? Maybe an opt-in for messages like this such as `IREE_DIAGNOSE_CONFIGURATION`?

Reply:
Actually, maybe `VERBOSE` is enough? https://cmake.org/cmake/help/latest/command/message.html I'll try it...

> `VERBOSE`
>     Detailed informational messages intended for project users. These messages should provide additional details that won’t be of interest in most cases, but which may be useful to those building the project when they want deeper insight into what’s happening.

Reply:
verbose would be nice - we can use that in other places too so the default case would be off and then if you hit issues turn it on - like one should be able to find on stack overflow (https://stackoverflow.com/questions/2670121/using-cmake-with-gnu-make-how-can-i-see-the-exact-commands)

Reply:
`VERBOSE` seems to work, _after a clean reconfigure_